packages/maskbook/src/plugins/RedPacket/schema.json

```diff
@@ -14,6 +10,10 @@
             "type": "string",
             "title": "rpid"
         },
+        "txid": {
```

Comment:
It is not breaking change, since `{ "additionalProperties": true }` and `"contract_version"` is removed from `"required"` field, old red-packet payload still works.